polaris-react/src/types.ts

```diff
@@ -407,7 +407,8 @@ export interface FilterInterface {
   hidden?: boolean;
 }
 
-/* Useful for defining mutually exclusive props such as:
+/**
+ * Useful for defining mutually exclusive props such as:
```

Comment:
Needed to trigger the GitHub workflow by changing something within `polaris-react/src/**`